dbt/models/default/default.vw_pin_value.sql

```diff
@@ -1,190 +1,472 @@
 -- View containing values from each stage of assessment by PIN and year
 -- in wide format
 
+-- Stage name constants
+{% set stage_name_pre_mailed = 'PRE-MAILED' %}
+{% set stage_name_mailed = 'MAILED' %}
+{% set stage_name_pre_certified = 'ASSESSOR PRE-CERTIFIED' %}
```

Comment:
This name feels a bit clunky to me, but it felt like the best option. Can you think of a better name for this stage?